tests/pytests/test_addons.py

```diff
@@ -99,6 +101,110 @@ def test_gdma():
     assert psi4.compare_matrices(totvals, ref_tot_mat, 6, "DMA Total Multipoles")
 
 
+@pytest.mark.smoke
+def test_ipi_broker1():
+    """ipi_broker1"""
+    from psi4.driver.broker import ipi_broker
+    water = psi4.geometry("""
+      O -1.216  -0.015  -0.261
+      H -1.946   0.681  -0.378
+      H -1.332  -0.754   0.283
+      units angstrom
+      no_reorient
+      no_com
+    """)
+
+    psi4.set_options({
+        'basis': 'sto-3g',
+        'reference': 'rhf',
+    })
+
+    options = {
+        "LOT"                   :   "ccsd",
+        "charge"                :   0,
+        "multiplicity"          :   1,
+    }
+
+    #ipi_broker(serverdata="inet:localhost:21340", options=options)
+    b = ipi_broker(serverdata=False, options=options)
```

Comment:
Would it be violating any of the conventions i-pi users are familiar with to convert to something like https://github.com/psi4/psi4/pull/1676/commits/403e742cab2fb8cd358d09df8e5e44c31aa7f97d#diff-c4d6e39b37f0195e3587302260658d00R504-R506 so that `energy(LOT, use_ipi=True, molecule=molecule)` still with an initialization step like https://github.com/psi4/psi4/pull/1676/files#diff-0fc31e8d2012cf40cdddc336b0b899ecR25 ?

Reply:
That would be fine too, I guess. I'm just not familiar enough with psi to directly implement it like this. Calling `energy(LOT, use_ipi=True, molecule=molecule)` along an initialization to provide the serverdata (see an example above commented out `serverdata="inet:localhost:21340"` to listen on the specific network interface/port) can be also used.
Where would the documentation belong into psi to ease the usage of the above convention from psi?

Reply:
I think I'll skip the `energy(..., ipi=True)` interface until there's an accustomed ipi user to plan with. Not following the "documentation belong into psi to ease" comment.

I'm content with the last 9ee06b1 commit. That'll fail the 2nd Azure lane until a fix for #1943 is in. After that, I'll rebase, then **force push** (so don't pull your branch again). CI will pass, then ready for merge.